nebula_ros/schema/sub/topic.json

```
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Topic parameters.",
+  "type": "object",
+  "definitions": {
+    "advanced_diagnostics": {
+      "type": "boolean",
+      "default": false,
+      "readOnly": true,
+      "description": "Enable advanced diagnostics."
+    },
+    "base_frame": {
+      "type": "string",
+      "default": "base_link",
+      "readOnly": true,
+      "description": "Frame with true sensor pose. The final sensor data is transformed by base_frame -> frame_id."
+    },
+    "diag_span": {
+      "type": "integer",
+      "default": 1000,
+      "minimum": 0,
```

`0 ms` timer period would not be any good, let's set it to a `> 0` value (`1 ms` is technically valid but still insane....)
```suggestion
      "minimum": 1,
```